bin/meshroom_photogrammetry

```diff
@@ -80,6 +80,8 @@ elif os.path.isfile(args.input) and os.path.splitext(args.input)[-1] in ('.json'
     # args.input is a sfmData file: setup pre-calibrated views and intrinsics
     from meshroom.nodes.aliceVision.CameraInit import readSfMData
     views, intrinsics = readSfMData(args.input)
+else:
```

Comment:
This is an error, as `args.inputImages` could have been used instead of `args.input`.

Reply:
New PR to fix it: https://github.com/alicevision/meshroom/pull/577